scripts/eslint/rules/no-ts-debug.cjs

```diff
@@ -0,0 +1,52 @@
+const { AST_NODE_TYPES, TSESTree } = require("@typescript-eslint/utils");
+const { createRule } = require("./utils.cjs");
+
+const tsNamespaceBarrelRegex = /_namespaces\/ts(?:\.ts|\.js|\.mts|\.mjs|\.cts|\.cjs)?$/;
+
+module.exports = createRule({
+    name: "no-ts-debug",
+    meta: {
+        docs: {
+            description: ``,
+        },
+        messages: {
+            importDebugError: `ts.Debug should be imported directly from src/compiler/debug.ts`,

```

Comment:
It'd be nice to have an auto fix for this, but I'd have to know where to put the import. Maybe just sticking it on the first line would be sufficient, and the lint can fix it later.